### ISSUE TITLE
Building with --fetch node fails

### ISSUE THREAD

**Issue by AtnNn** [rethinkdb/rethinkdb#4022]
In a clean tree. In next.

```
mk/support/pkg/npm-pkg.inc: line 11: build/external/node_0.10.15/bin/npm: No such file or directory
```

**Comment by danielmewes** [comment]
We should consider fixing this in 2.0 if we have time for it.

**Comment by AtnNn** [comment]
That would be impossible. It is not broken in 20.

**Comment by danielmewes** [comment]
Ah got it. Wouldn't make sense to backport the bug I guess. :)

**Comment by AtnNn** [comment]
I've fixed this bug locally, but it reveals another problem: our web assets don't build with npm 1.3.5. Errors like this one occur:

```
error Error: No compatible version found: lodash._basetostring@'^3.0.0'
```

It'll take a little more work than I thought to get `--fetch node` working again.

**Comment by AtnNn** [comment]
Fixed in 61e53ed, with help from #3687
